include/benchmark/benchmark.h

```diff
@@ -450,6 +450,8 @@ enum BigO { oNone, o1, oN, oNSquared, oNCubed, oLogN, oNLogN, oAuto, oLambda };
 
 typedef uint64_t IterationCount;
 
+enum StatisticUnit { Time, Percentage };
```

Comment:
https://google.github.io/styleguide/cppguide.html#Enumerator_Names

`kTime` and `kPercentage` please